apps/Open-Typer/install-32

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+version=5.2.0
+
+#Install Depends
+install_packages libfuse2 || exit 1
+
+#Download Open-Typer build from repo
+wget -O /tmp/Open-Typer.AppImage https://github.com/Open-Typer/Open-Typer/releases/download/v${version}/Open-Typer-armhf.AppImage || error "Failed to download AppImage!"
+
+#Move Open-Typer.AppImage to /opt
+sudo mv /tmp/Open-Typer.AppImage /opt || error "Failed to move from /tmp to /opt!"
+
+#Mark Open-Typer.AppImage as executable
+sudo chmod +x /opt/Open-Typer.AppImage || error "Failed to mark as executable!"
+
+#Create Menu Button
+status "Creating Menu Button"
+sudo mkdir -p /usr/local/share/applications
+echo "[Desktop Entry]
+Name=Open-Typer
+Comment=An Open Source spiritual successor to the flash game Line Rider
+Icon=$(dirname "$0")/icon-64.png
+Exec=/opt/Open-Typer.AppImage %f
+Type=Application
+Terminal=false
+Categories=Utility;" | sudo tee /usr/local/share/applications/Open-Typer.desktop >/dev/null || error "Failed to create menu button!"
```

Comment:
- Are you sure Utility is the best category for this? I was thinking Education would be the best fit.
- Please update the comment.
- The `%f` argument in the Exec line denotes the ability to open files. Does Open-Typer actually have the ability to open any kind of files?

Reply:
Meant to change Utility, sorry. Did not know how to execute an appimage from a menu entry, and saw both Cura and OpenSCAD use it, so I put it in. You can remove it.

Reply:
Any idea what %U does then? adazem009 thought it was also files.

Reply:
A quick search told me it was for accessing urls. 
![image](https://github.com/Botspot/pi-apps/assets/98842253/59e84402-64db-46ea-b0b7-dcec3c88a3d4)